test/e2e/default_bridged_network_test.go

```diff
@@ -90,9 +90,13 @@ var _ = Describe("NodeNetworkConfigurationPolicy default bridged network", func(
 		})
 
 		It("should successfully move default IP address on top of the bridge", func() {
+			Skip("Cluster does not comes back")
 			By("Creating the policy")
 			setDesiredStateWithPolicy("default-network", createBridgeOnTheDefaultInterface)
 
+			By("Waiting until the node becomes ready again")
```

Comment:
Why do we have all these changes on the test if we skip? I think we should not change test that is not being ran.

Reply:
I think I merged changes to debug it, do I need to discard them ?

Reply:
Meh, not a big deal.